aospy/automate.py

```diff
@@ -22,6 +22,7 @@
 _REGIONS_STR = 'regions'
 _VARIABLES_STR = 'variables'
 _TAG_ATTR_MODIFIERS = dict(all='', default='default_')
+_TIME_DEFINED_REDUCTIONS = ['av', 'std', 'ts', 'reg.av', 'reg.std', 'reg.ts']
```

Comment:
For now I think it might be a good idea to keep the definition of these only in `calc.py` and import them here.  This way we won't need to remember to maintain this list in two places. 

```python
from calc import _TIME_DEFINED_REDUCTIONS
```